doc/sphinx/msgtypes.rst

```diff
@@ -0,0 +1,31 @@
+Message Types
+=============
+
+.. glossary::
+
+   emergency_stop
+      Boolean if Emergency STOP was pressed. Should be reported on system start
+      and on change.
+
+   encoders
+      List of differences of all available encoders since the last update. For two
+      wheeled robot they shell be ordered [left, right]. Positive difference
+      corresponds to forward motion. The scaling factor is specific to robot
+      and can change over time (preasure in tires, payload). The main purpose
+      of this message type is an advanced position estimation.
+
+   gps_position
+      [longitude_ms, latitude_ms] - GPS coordinates in WGS84 system. Scaled
+      3600000 (i.e. in milliseconds)
+
+   pose2d
+      [x, y, heading] - position in meters with heading in radians
+      (anticlockwise). Scale 1000x (i.e. x, y are send in millimeters)
+      and angle in 1/100th of degree.
```

Comment:
I don't understand "heading in radians" vs "angle in 1/100th of degree" while the transformation should be "Scale 1000x".

Reply:
I removed the scaling factor as "implementation detail" which we want to hide anyway.

Reply:
You removed it from the description but the actual messages used in the code are still integers with the scaling, right? So what is the plan with that? I am confused now. I propose that the documentation follows what is implemented in contrary to what might be implemented in the future.

```suggestion
pose2d
      [x, y, heading] - position in millimeters with heading in 1/100th of degree.
```

Reply:
OK, now there is a note about scaling factor, thanks.